theme/static/style/style.css

```diff
@@ -521,10 +561,23 @@ pre code {
         padding-right: 1.8rem;
     }
 
-    /* screenshot section */
-    .screenshots .screenshot-title {
+    /* screenshot section and features page */
+    .screenshots .screenshot-title, .features .screenshot-title {
         text-align: left;
         padding-left: 1.8rem;
         box-sizing: border-box;
     }
+
+    /*features page*/
+    .features .w50 {
+        width: 100%;
+    }
+
+    .features .col-8 li {
```

Comment:
remove this block and bring back the definition for horizontally stacked list which has been deleted but it should be present.
```
.features .col-8 .horizontally-stacked-list {
        display: inline;
    }
```